tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,32 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## July 11, 2023
+
+**General changes**
+
+- [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) now is Generally Available.
+
+- Introduce TiDB Bot (beta), an OpenAI-powered chatbot that offers multi-language support, 24/7 real-time response, and integrated documentation access.
+
+    TiDB Bot provides you with the following benefits:
+
+    - Continuous support: Always available to assist and answer your questions for an enhanced support experience.
+    - Improved efficiency: Automated responses reduce latency, improving overall operations.
+    - Seamless documentation access: Direct access to TiDB Cloud documentation for easy information retrieval and quick issue resolution.
```

Comment:
[ref](https://developers.google.cn/style/colons#colons-within-sentences): In general, the first word in the text that follows a colon should be in lowercase.
```suggestion
    - Continuous support: always available to assist and answer your questions for an enhanced support experience.
    - Improved efficiency: automated responses reduce latency, improving overall operations.
    - Seamless documentation access: direct access to TiDB Cloud documentation for easy information retrieval and quick issue resolution.
```